### PR TITLE
add doc for rootTocPath

### PR DESCRIPTION
**Pull request by superyyrrzz**
#908 
@vwxyzh @vicancy @hellosnow @ansyral @qinezh @chenkennt

### CODE REVIEW

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 1]
```diff
@@ -407,6 +407,7 @@ exclude            | The files to be excluded, `glob` pattern is supported.
 src                | Specifies the source directory. If omitted, the directory of the config file will be used. Use this option when you want to refer to files in relative folders while want to keep folder structure. e.g. set `src` to `..`.
 dest               | The folder name for the generated files.
 version            | Version name for the current file mapping. If not set, treat the current file-mapping item as in default version. Mappings with the same version name will be built together. Cross reference doesn't support cross different versions.
+rootTocPath        | The Root TOC Path used for navbar in current version, relative to output root. If not set, will use the TOC in output root in current version if exists. It's useful for a version to specify a navbar TOC no matther whether it's within the same version.
```
Comment (hellosnow) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >used [](start = 39, length = 4)

'is' used for?

**Comment by hellosnow** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

File: Documentation/tutorial/docfx.exe_user_manual.md (hunk 1 above)
Comment (vicancy) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >version [](start = 66, length = 7)

aslo introduce the concept of `version`?

Comment (vicancy) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >rootTocPath [](start = 0, length = 11)

why is it in file mapping? wrong place?

**Comment by superyyrrzz** [comment]
close this as may be refactored later...

**Pull request closed without merging**